app/src/main/res/values/styles.xml

```diff
@@ -826,4 +826,90 @@
     <item name="endIconDrawable">@drawable/ic_arrow_drop_down_black_24dp</item>
     <item name="endIconTint">@color/component_color_shared_black_background_color</item>
   </style>
+
+  <style name="WalkthroughFinalNoTextViewStyle" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">16sp</item>
+    <item name="android:textColor">@color/component_color_shared_primary_text_color</item>
+  </style>
+
+  <style name="WalkthroughFinalYesTextViewStyle" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif</item>
+    <item name="android:textSize">20sp</item>
+    <item name="android:textColor">@color/component_color_shared_secondary_6_text_color</item>
+  </style>
+
+  <style name="DeveloperOptionsTextViewStyle" parent="TextView.Common1">
+    <item name="android:fontFamily">sans-serif-medium</item>
+    <item name="android:textSize">14sp</item>
+    <item name="android:textColor">@color/component_color_shared_primary_dark_text_color</item>
+  </style>
+
+    <style name="OnboardingLanguageTextViewStyle" parent="TextView.Common1">
+      <item name="android:layout_width">wrap_content</item>
+      <item name="android:layout_height">wrap_content</item>
+      <item name="android:textSize">16sp</item>
+      <item name="android:textColor">@color/component_color_shared_primary_text_color</item>
+      <item name="android:fontFamily">sans-serif</item>
+      <item name="android:padding">8dp</item>
+    </style>
+
+    <style name="WalkthroughFinalTitleTextViewStyle" parent="TextView.Common1">
+      <item name="android:layout_width">wrap_content</item>
+      <item name="android:layout_height">wrap_content</item>
+      <item name="android:textSize">24sp</item>
+      <item name="android:textColor">@color/component_color_shared_primary_text_color</item>
+      <item name="android:fontFamily">sans-serif</item>
+    </style>
+
+    <style name="ChapterIndexTextViewStyle" parent="TextView.Common1">
```

Comment:
This style is used in 4 places, but it contains some attributes that dont have the same values for the different screens, e.g `minWidth` and `minHeight`, as well as `padding` values, are not consistent. The style should be trimmed down to what is similar accross all 4 views, and whatever is different should be maintained directly in the view.